### ISSUE TITLE
The training loss has been around 6.9

### ISSUE THREAD

**Issue by FuHTong** [caiocj1/vit-learning#1]
Hello, I have encountered the situation that the loss in model training has been hovering around 6.9, using the vit model built by myself. Similarly, the same thing happened when I used your project for training, but it was normal to load vit_b_16(), and the data set used was ILSVRC2012. Have you ever had such a situation? Can you give me some advice on model building and training? Thank you very much

**Comment by FuHTong** [comment]
> Hello, I have encountered the situation that the loss in model training has been hovering around 6.9, using the vit model built by myself. Similarly, the same thing happened when I used your project for training, but it was normal to load vit_b_16(), and the data set used was ILSVRC2012. Have you ever had such a situation? Can you give me some advice on model building and training? Thank you very much
Well, there should be a problem with my code, your project can slowly reduce the loss, I will learn again, thank you for your open source code

**Comment by FuHTong** [comment]
> Hello, I have encountered the situation that the loss in model training has been hovering around 6.9, using the vit model built by myself. Similarly, the same thing happened when I used your project for training, but it was normal to load vit_b_16(), and the data set used was ILSVRC2012. Have you ever had such a situation? Can you give me some advice on model building and training? Thank you very much

Well, there should be a problem with my code, your project can slowly reduce the loss, I will learn again, thank you for your open source code
